src/descriptor/create_descriptor.rs

```diff
@@ -39,7 +41,7 @@ fn parse_scriptsig_top<'txin>(
     script_sig: &'txin bitcoin::Script,
 ) -> Result<(Vec<u8>, Stack<'txin>), Error> {
     let stack: Result<Vec<StackElement>, Error> = script_sig
-        .iter(true)
+        .instructions_minimal()
         .map(|instr| instr_to_stackelem(&instr))
```

Comment:
Why not simply `.map(instr_to_stackelem)`? I tried locally, it compiles well with all versions, starting from MSRV

Reply:
I think this is unrelated to the intent of the PR. It is indeed a useful simplification. Can you raise another PR for this?

Reply:
Done: https://github.com/rust-bitcoin/rust-miniscript/pull/133